sdk/cosmos/azure-cosmos/azure/cosmos/_synchronized_request.py

```diff
@@ -241,6 +269,29 @@ def SynchronizedRequest(
     elif request.data is None:
         request.headers[http_constants.HttpHeaders.ContentLength] = 0
 
+    if request_params.availability_strategy_config is None:
+        # if ppaf is enabled, then hedging is enabled by default
+        if global_endpoint_manager.is_per_partition_automatic_failover_enabled():
+            request_params.availability_strategy_config = CrossRegionHedgingStrategyConfig()
+
+    # Handle hedging if availability strategy is applicable
+    if _is_availability_strategy_applicable(request_params):
+        return _availability_strategy_handler.execute_with_hedging(
```

Comment:
is there any advantage or reason for importing this here at runtime as opposed to just importing it by default up top? I would think there's no need since the file is not extremely big or anything